desktop/apps/categories2/stylesheets/index.styl

```diff
@@ -39,13 +39,6 @@
 }
 
 .categories2-genes {
-	display flex
-	flex 0 1 auto
-	flex-flow row wrap
-	justify-content flex-end
+	column-count 3
 	padding: 1em 0
```

Comment:
I guess it would be more Stylus-y to leave the `:` out. 

Maybe we should have a column gutter here as well, e.g. `column-gap: 1em`